container_engine/django_tutorial/Dockerfile

```diff
@@ -21,11 +21,11 @@ FROM gcr.io/google_appengine/python
 
 # Create a virtualenv for the application dependencies.
 # # If you want to use Python 3, add the -p python3.4 flag.
-RUN virtualenv /env
+RUN virtualenv -p python3.4 /env
```

Comment:
I bumped this to 3.6 and confirmed it works 👌

Reply:
Thanks. I changed the version.